src/unix/linux_like/linux/mod.rs

```diff
@@ -597,6 +624,97 @@ cfg_if! {
             }
         }
 
+        #[cfg(libc_union)]
```

Comment:
We could use `cfg_if!` macro here as well.

Reply:
fixed